src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,54 @@ However, we have retracted this method for the following reasons:
 * Various studies, such as [Baker et al.](https://www.acpjournals.org/doi/10.7326/M20-7145) have shown that, for at least some pockets of healthcare workers, healthcare workers are no more likely to get infected through their work than the average person is to get infected through socializing.
 * The above Washington State report shows that people who work outside the home got COVID at about the average rate for the state.
 
+# Vaccines Q&A
+
+## How should we think about people who have been vaccinated?
+
+There are two questions that are important for understanding vaccines:
+
+1. How much safer for the vaccinated individual is exposure to COVID?
+2. How much less likely is it for a vaccinated individuals to expose others to COVID? (Or equivalently, how much less likely is a vaccinated individual to catch COVID?)
+
+We address these two questions individually.
+
+### 1. How much safer for the vaccinated individual is exposure to COVID?
+All currently approved vaccines are proven significantly reduce the chance of getting sick. If you do get sick after being vaccinated, you are significantly less likely to be hospitalized or die. However, one of the scary things about COVID is that cases with no outward symptoms have been found to cause severe lung damage [source?]. As a result:
+
+- For people who are trying to avoid COVID to avoid long term health problems, we do not recommend changing your budget (beyond what the reduction in microCOVIDs from #2 allows you to do).
+- For people who are trying to avoid COVID to avoid hospitalization or death as a direct result of COVID (generally those who are more vulnerable to COVID), having both doses of the AstraZeneca, Pfizer, or Moderna vaccines and waiting 1-2 weeks after the second dose should reduce your chance of death by an additional 2-10x over the microCOVID reduction outlined in part 2. As such, you may consider increasing your microCOVID budget from 0.1% / year to somewhere between 0.2% / year and 1% / year.
+
+
+### 2. How much less likely is it for vaccinated individuals to expose others to COVID?
+
+We do not know for sure how likely it is for a vaccinated individual to expose others to COVID. Vaccinated individuals still have a chance of becoming sick (even if they are not seriously ill), and may be asymptomatic carriers of the virus, capable of infecting others without having symptoms themselves. None of the vaccine studies have directly addressed what the chances of transmission from a vaccinated individual are. We attempt to make educated guesses by pulling in other sources. For our derivation of these numbers, see the [Research Sources](/paper/14-research-sources#vaccines)
```

Comment:
The link is there: `#vaccines`. I think this might be the bug where jumping to headers in links doesn't consistently work?

Reply:
Oh I must 

> You have to be really precise with what you mean by "whatever it would have been." It's *.2 what it would have been if you were doing the same things and recording them as if you were unvaccinated. This kind of critical, vaccination possibly blows a huge hole in basic method people; I'd expect a vaccinated person to start taking more risks, which means they wouldn't actually be 5x safer.
> 
> In the Risk Tracker, you'd model them as vaccinated custom people, which would make them 5x safer than unvaccinated custom people doing the same thing.

Got it. So I think there should be some mention of that in this Q&A section. As a user of the calculator or the risk tracker, a big question I have is "how does it change the risk of an activity if the person I"m seeing is vaccinated?" → And then your answer can say something like: if you're using the Risk Tracker to model their activities, or you're using one of the risk profiles like "closed pod of X," then you get the full 5x bonus. But for average folks in your area, they don't get the full 5x bonus because they might actually be taking more risk after their vaccination.

(That's an un-wordsmithed version.)